app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -50,6 +50,26 @@
       %span.help-block{"ng-show" => "angularForm.#{prefix}_api_port.$error.detectedSpaces"}
         = _("Spaces are prohibited")
 
+%div{"ng-if" => defined?(database_name_show) ? true : false}
+  .form-group{"ng-class" => "{'has-error': angularForm.#{prefix}_database_name.$invalid}",
+    "ng-if" => "emsCommonModel.emstype == 'rhevm'"} |
```

Comment:
if I remove this ng-if it shows in the "Default" (host) pane and it should not be there.